docs/v3/tutorials/alerts.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Send alerts to Slack on failure
```

Comment:
Slack alerts are certainly very common, but starting with an email alert might be a better choice as it requires no 3rd party setup. Plus you can send emails directly to a Slack channel to circumvent generating a token.

Reply:
https://github.com/PrefectHQ/prefect/pull/16345/commits/ae06a7a2f8039f03f9cccd716609db113ee40831